lib/overcommit/git_repo.rb

```diff
@@ -20,8 +20,9 @@ def extract_modified_lines(file_path, options)
       lines = Set.new
 
       flags = '--cached' if options[:staged]
+      refs  = 'HEAD~ HEAD' if options[:last_commit]
```

Comment:
I don't see the need to limit the utility of this option to only allow retrieving the last commit. Let's change it to `options[:refs]` for now and pass it directly.